cmd/nerdctl/version.go

```diff
@@ -66,7 +67,17 @@ func versionAction(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	v, vErr := versionInfo(cmd, globalOptions)
+	address := globalOptions.Address
+	// rootless `nerdctl version` runs in the host namespaces, so the address is different
+	if rootlessutil.IsRootless() {
+		address, err = rootlessutil.RootlessContainredSockAddress()
+		if err != nil {
+			logrus.WithError(err).Warning("failed to inspect the rootless containerd socket address")
+			address = ""
```

Comment:
`address = ""` is useless

Reply:
This is added because a future version of `RootlessContainredSockAddress` may potentially return a non-empty address with a non-nil err.

Reply:
Why ?